client/src/components/RelatedContainer.jsx

```diff
@@ -9,23 +9,47 @@ export default function RelatedContainer () {
   //related data needed. {category,name,img,star,def price, sale price }
 
   //style data need to search styles find default? to be true and use that data...
-  const [productInfo, setProductInfo] = useState([1,2,3])//DummyData set to 3 indexes
-
+  const [productsInfo, setProductsInfo] = useState([1,2,3])//DummyData set to 3 indexes
+  const dummyId = 1;
```

Comment:
I would recommend adding a hook for `[ productId, setProductId ]` as a placeholder, and set its default value with useState. If I understand correctly, we'll want all the child components to re-render once the App selects a product... so we'll all need the components to hook into the productId state in App, right?